### PR TITLE
Add ndindex

### PR DESCRIPTION
**Pull request by asmeurer**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by asmeurer** [comment]
@scopatz

File: recipes/ndindex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  imports:
+    - ndindex
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/ndindex/
+  summary: A Python library for manipulating indices of ndarrays.
```
Comment (synapticarbors) [inline recipes/ndindex/meta.yaml:37]: Add a comment linking to the following PR as a reminder to potentially use the upstream license file for future releases:

https://github.com/Quansight/ndindex/pull/2

File: recipes/ndindex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  license: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/ndindex/meta.yaml:39]: Add `dev_url: https://github.com/Quansight/ndindex`

File: recipes/ndindex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
```
Comment (scopatz) [inline recipes/ndindex/meta.yaml:40]: Also, please add `license_family: MIT`

**Pull request merged**